<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ttlvConvertJson</class>
 <widget class="QWidget" name="ttlvConvertJson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget{
background-color: rgb(255, 255, 255);
border-style:solid;
border-width:1px;
border-color:rgb(240,240,240);
border-radius:4px;
}
QScrollBar:vertical{ 
    width:8px;  
    border-style:flat;
    border-radius: 4px;
    border:0px;
    background: rgb(255, 255, 255);
} 
QScrollBar::handle:vertical{ 
    background: rgb(230, 230, 230);
    border-radius: 4px;
    width:8px; 
    min-height:50px; 
    border-style:flat;
}
QScrollBar::handle:vertical::hover{ 
    background: rgb(200, 200, 200);
    border-radius: 4px;
    width:8px; 
}
QScrollBar::handle:vertical::pressed{ 
    background: rgb(200, 200, 200);
    border-radius:4px;
    width:8px; 
}
QScrollBar::sub-page:vertical {
     background: rgb(255, 255, 255);
border-style:flat;
}
QScrollBar::add-page:vertical {
     background: rgb(255, 255, 255);
border-style:flat;
}
QScrollBar::add-line:vertical{
     background: rgb(255, 255, 255);
}
QScrollBar::sub-line:vertical {
     background: rgb(255, 255, 255);
}
QScrollBar:horizontal{ 
    height:8px;  
    border-style:flat;
    border-radius: 4px;
    border:0px;
    background: rgb(255, 255, 255);
} 
QScrollBar::handle:horizontal{ 
    background: rgb(230, 230, 230);
    border-radius: 4px;
    height:8px; 
    min-width:50px; 
    border-style:flat;
}
QScrollBar::handle:horizontal::hover{ 
    background: rgb(200, 200, 200);
    border-radius: 4px;
    height:8px; 
}
QScrollBar::handle:horizontal::pressed{ 
    background: rgb(200, 200, 200);
    border-radius:4px;
    height:8px; 
}
QScrollBar::sub-page:horizontal {
     background: rgb(255, 255, 255);
    border-style:flat;
}
QScrollBar::add-page:horizontal {
     background: rgb(255, 255, 255);
    border-style:flat;
}
QScrollBar::sub-line:horizontal {
     background: rgb(255, 255, 255);
}
QScrollBar::add-line:horizontal{
     background: rgb(255, 255, 255);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_windowTitle" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent; </string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,1">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_windowTitle">
             <property name="toolTip">
              <string/>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight:bold;
font-size:16px;
color: rgb(50, 50, 50);
</string>
             </property>
             <property name="text">
              <string>TTLV - JSON 工具</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>623</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_Minimize">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover { 
background-color: rgb(220, 220, 220); 
}
QPushButton { 
image: url(:/png/Minimize.png);
border-width:0;
border-style:outset;
background-color:transparent;
border-radius:2px;
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Maximize">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover { 
background-color: rgb(220, 220, 220); 
}
QPushButton { 
image: url(:/png/MaximizeOff.png);
border-width:0;
border-style:outset;
background-color:transparent;
border-radius:2px;
}
QPushButton:disabled
{
border-color:rgb(240, 240, 240);
border-radius:4px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_close">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover { 
background-color:rgb(242, 113, 113); 
image: url(:/png/closeW.png)15 0 0 0;
}
QPushButton { 
image: url(:/png/close.png)15 0 0 0;
border-width:0;
border-style:outset;
background-color:transparent;
border-radius:2px;
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(200, 200, 200);</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: rgb(47, 105, 235);
	font-size:14px;
	border-style:solid;
	border-width:1px;
	border-color:rgb(47, 105, 235);
	border-radius:4px;
}
QPushButton:hover{
	color: rgb(47, 105, 235);
	background-color: rgba(47, 105, 235, 0.08);
}
QPushButton:pressed{
	color: rgb(47, 105, 235);
	background-color: rgba(47, 105, 235, 0.2);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,0,3">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QTextEdit" name="textEdit_src">
           <property name="styleSheet">
            <string notr="true">color: rgb(104, 112, 130);
font-size:14px;
border-style:solid;
border-width:1px;
border-height：20px;
border-color:rgb(229, 231, 234);
border-radius:4px;
padding-left:5px;
padding-right:5px;

</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioButton_TJ">
             <property name="styleSheet">
              <string notr="true">QRadioButton::indicator{
    width: 30px;
    height: 23px;
}

QRadioButton::indicator:unchecked{
    image:url(:/png/checkBox0.png);
}

QRadioButton::indicator:checked{
    image:url(:/png/checkBox1.png);
}
QRadioButton{background-color:transparent;color: rgb(104, 112, 130);}
</string>
             </property>
             <property name="text">
              <string>ttlv转json</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_JT">
             <property name="styleSheet">
              <string notr="true">QRadioButton::indicator{
    width: 30px;
    height: 23px;
}

QRadioButton::indicator:unchecked{
    image:url(:/png/checkBox0.png);
}

QRadioButton::indicator:checked{
    image:url(:/png/checkBox1.png);
}
QRadioButton{background-color:transparent;color: rgb(104, 112, 130);}
</string>
             </property>
             <property name="text">
              <string>json转ttlv</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_change">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>36</height>
              </size>
             </property>
             <property name="text">
              <string>确认</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="styleSheet">
            <string notr="true">QHeaderView::section{
color: rgb(104, 112, 130);
font-size:14px;
background: transparent;
background: rgb(246, 249, 255);
padding: 1px;border: none;
border-style:solid;
border-right-width:0px;
border-left-width:0px;
}
QTableWidget{
color: rgb(104, 112, 130);
font-size:14px;
border-style:solid;
border-width:1px;
border-color:rgb(229, 231, 234);
border-radius:4px;
}</string>
           </property>
           <column>
            <property name="text">
             <string>源格式</string>
            </property>
            <property name="font">
             <font>
              <pointsize>25</pointsize>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>源数据</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>转换后数据</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>备注</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
